File: app/src/main/java/com/kevin/kggank/ui/view/IGanhuoDetailView.java
package com.kevin.kggank.ui.view;

import com.kevin.kggank.base.IBaseView;
import com.kevin.kggank.entity.GanhuoOneDayEntity;

/**
 * Created by kevin on 16/5/23.
 */

public interface IGanhuoDetailView extends IBaseView {

    public void getGanhuoOneDay(GanhuoOneDayEntity.GanhuoOneDayDataEntity ganhuoOneDayDataEntity);
}

File: app/src/main/java/com/kevin/kggank/ui/view/IGanhuoDetailView.kt
package com.kevin.kggank.ui.view

import com.kevin.kggank.entity.GanhuoOneDayEntity
import com.kevin.kglib.base.LibIBaseView

/**
 * Created by kevin on 16/5/23.
 */

interface IGanhuoDetailView : LibIBaseView {

    fun getGanhuoOneDay(ganhuoOneDayDataEntity: GanhuoOneDayEntity.GanhuoOneDayDataEntity)
}
